go.mod

```diff
@@ -41,9 +44,10 @@ require (
 	github.com/stretchr/objx v0.1.1 // indirect
 	github.com/stretchr/testify v1.3.0
 	golang.org/x/crypto v0.0.0-20190513172903-22d7a77e9e5f
-	golang.org/x/net v0.0.0-20190514140710-3ec191127204 // indirect
-	golang.org/x/sys v0.0.0-20190514135907-3a4b5fb9f71f // indirect
+	golang.org/x/net v0.0.0-20190522155817-f3200d17e092 // indirect
```

Comment:
do you know if every run of go mod vendor updates these dependencies? should we pin them?

Reply:
I don't think so. We don't use it explicitly, hence `indirect`, and `gomodvendor` does not pin it.

Reply:
it is just strange that they got updated, probably one of the new indirect deps have this in their go.mod file then. but take a look at those deps.. i don't think that it is correct to add them

Reply:
It's being updated whenever you do `go get -u some/module` and that module depends on it. It is not pinned anywhere so it gets updated to the latest. Also this package does not support go modules so that's why nothing pins it down.

Reply:
so after you removed go-bindata and modvendor from go.mod this dependency is still updated. is it a side effect that wasn't reverted by a go mod tidy? not a big deal, just trying to understand why it happened